operator/pkg/karmadaresource/apiservice/apiservice.go

```diff
@@ -30,8 +30,8 @@ func init() {
 }
 
 // EnsureAggregatedAPIService creates aggregated APIService and a service
-func EnsureAggregatedAPIService(aggregatorClient *aggregator.Clientset, client clientset.Interface, name, namespace, caBundle string) error {
-	if err := aggregatedApiserverService(client, name, namespace); err != nil {
+func EnsureAggregatedAPIService(aggregatorClient *aggregator.Clientset, client clientset.Interface, name, namespace, karmadaControlPlaneServiceName, karmadaControlPlaneNamespace, caBundle string) error {
```

Comment:
So， if I deploy following yaml:
```
apiVersion: operator.karmada.io/v1alpha1
kind: Karmada
metadata:
  name: karmada-demo
  namespace: test
```

The parameter is named "karmada-demo" and the parameter namespace is "test". Why do we need it as a parameter? Can you explain it to me?

Reply:
<img width="957" alt="image" src="https://github.com/karmada-io/karmada/assets/7907809/249fdb56-f949-446e-b98b-9b0838ec481e">

karmada instance will create a constant namespace karmada-system to save karmada resources, including aggregated service

Reply:
I understand, thank you for explaining.

So can we make the parameter name more unified, one is host cluster, one is karmada control plane.

Here, the demo ns is in host cluster, karmada-system is in karmada-control plane.


So the parameter should be: 
```
func EnsureAggregatedAPIService(aggregatorClient *aggregator.Clientset, client clientset.Interface, karmadaServiceName, KarmadaServiceNamespace, hostClusterServiceName, hostClusterNamespace, caBundle string)
```

@XiShanYongYe-Chang, please help me confirm.

Reply:
It's a bit too hard to name, before the logic in our code and in the documentation we advertised to the public, one was `host cluster` and the other was `karmada control plane`.

Reply:
> It's a bit too hard to name, before the logic in our code and in the documentation we advertised to the public, one was `host cluster` and the other was `karmada control plane`.

Maybe this is more in line with karmada norms ? 

```golang
func EnsureAggregatedAPIService(aggregatorClient *aggregator.Clientset, client clientset.Interface, karmadaControlPlaneServiceName, karmadaControlPlaneNamespace, hostClusterServiceName, hostClusterNamespace, caBundle string)
```

Reply:
+1